live-examples/css-examples/fonts/font-variant-caps.css

```diff
@@ -0,0 +1,13 @@
+@font-face{
```

Comment:
Nit, code style: I reckon you just overlooked this one 👁

Reply:
Yeah I did. I just pushed a change to find-and-replace the problem everywhere. :-)